dev-requirements.txt

```diff
@@ -2,4 +2,6 @@ pypiwin32
 Pillow==3.0.0
 coverage
 nose
-codecov
\ No newline at end of file
+codecov
+rst2pdf
+Sphinx
```

Comment:
The file name is **dev-**requirements.txt. :) This is my experiments with pdf format of the docs. Sphinx html docs is a build artifact now. Pdf docs still fail.